components/brave_wallet/browser/zcash/rust/lib.rs

```diff
@@ -213,11 +188,11 @@ mod ffi {
         addr: [u8; 43],
         memo: [u8; 512],
         // Whether to use memo field in the transacition
-        use_memo: bool
+        use_memo: bool,
```

Comment:
it's really hard to tell what's changing here because of the formatting changes. Why are these happening? Are we not running clang format on rust code? @cypt4

Reply:
What is changed there is that I removed unused includes, however formatting changed considerably once I ran the automatic formatting.

Reply:
@cdesouza-chromium what did you run that changed the formatting? Per my other comment can we leave the formatting changes out for a follow-up so it's easier to see the changes?

Reply:
I've separated this into 93accacf9e86de2c07c3e3164bd6f49e140755a0. I usually use `git cl format` to format the code, which should be supported overall. I was rather surprised the whole file is getting affected, as `git cl format` only applies to the changed lines.